engine/installation/linux/docker-ee/ubuntu.md

```diff
@@ -0,0 +1,270 @@
+---
+description: Instructions for installing Docker EE on Ubuntu
+keywords: requirements, apt, installation, ubuntu, install, uninstall, upgrade, update
+redirect_from:
+- /engine/installation/ubuntulinux/
+- /installation/ubuntulinux/
+- /engine/installation/linux/ubuntulinux/
+title: Get Docker EE for Ubuntu
+---
+
+{% assign minor-version = "17.03" %}
+
+To get started with Docker EE on Ubuntu, make sure you
+[meet the prerequisites](#prerequisites), then
+[install Docker](#install-docker).
+
+## Prerequisites
+
+Docker CE users should go to
+[Get docker CE for CentOS](/engine/installatioin/linux/docker-ce/centos.md)
+**instead of this topic**.
+
+To install Docker Enterprise Edition (Docker EE), you need to know the Docker EE
+repository URL associated with your trial or subscription. To get this information:
+
+- Go to [https://store.docker.com/?overlay=subscriptions](https://store.docker.com/?overlay=subscriptions).
+- Choose **Get Details** / **Setup Instructions** within the
+  **Docker Enterprise Edition for Ubuntu** section.
+- Copy the URL from the field labeled
+  **Copy and paste this URL to download your Edition**.
+
+Use this URL when you see the placeholder text `<DOCKER-EE-URL>`.
+
+To learn more about Docker EE, see
+[Docker Enterprise Edition](https://www.docker.com/enterprise-edition/){: target="_blank" class="_" }.
+
+### OS requirements
+
+To install Docker EE, you need the 64-bit version of one of these Ubuntu versions:
+
+- Yakkety 16.10
+- Xenial 16.04 (LTS)
+- Trusty 14.04 (LTS)
+
+Docker EE is supported on `x86_64` architectures.
+
+### Uninstall old versions
+
+Older versions of Docker were called `docker` or `docker-engine`. In addition,
+if you are upgrading from Docker CE to Docker EE, remove the Docker CE package.
+
+```bash
+$ sudo apt-get remove docker docker-engine docker-ce
```

Comment:
we may want to add `docker.io` here, as there is a package named like that in the ubuntu repositories https://packages.ubuntu.com/trusty/docker.io